tests/test_port_add_remove.py

```diff
@@ -79,7 +79,7 @@ def test_remove_add_remove_port_with_buffer_cfg(self, dvs, testlog):
         config_db.delete_entry('PORT', PORT_A)
         num = asic_db.wait_for_n_keys("ASIC_STATE:SAI_OBJECT_TYPE_PORT",
                               num_of_ports-1,
-                              polling_config = PollingConfig(polling_interval = 1, timeout = 5.00, strict = True))
+                              polling_config = PollingConfig(polling_interval = 1, timeout = 30.00, strict = True))
```

Comment:
Why increase timeout?

Reply:
The timeout duration was increased after analyzing the failures and identifying that the issue might be related to initialization timing. Adding a delay helped resolve the problem, and the build was successful. However, when these delays were removed, the dvs_test failed again, indicating that the delays are necessary for the DVS to be ready.

Reply:
Why does this change need initialization time to be increased? Can you reduce this timeout to only what is needed?